src/client.rs

```diff
@@ -1250,6 +1250,36 @@ impl<T: Read + Write + Unpin + fmt::Debug + Send> Session<T> {
         Ok(c)
     }
 
+    /// The [`GETMETADATA` command](https://datatracker.ietf.org/doc/html/rfc5464.html#section-4.2)
+    pub async fn get_metadata(
+        &mut self,
+        mailbox_name: &str,
+        options: Option<&str>,
```

Comment:
What would `Some("")` mean f.e.? Maybe make it just `&str` with "" meaning no options?